fabric-network/src/impl/wallet/hsmx509identity.ts

```diff
@@ -14,14 +14,16 @@ export interface HsmX509Identity extends Identity {
 	type: 'HSM-X.509';
 	credentials: {
 		certificate: string;
+		privateKey: string;
 	};
 }
 
 interface HsmX509IdentityDataV1 extends IdentityData {
 	type: 'HSM-X.509';
-	version: 1;
+	version: 2;
```

Comment:
If it's representing v2 data then the interface name should be changed to `HsmX509IdentityDataV2`. But you will want to keep the `HsmX509IdentityDataV1` interface definition so that v1 format data can be cast to that type and read. So you should have both `HsmX509IdentityDataV1` and `HsmX509IdentityDataV2` interfaces.